src/main/java/bio/terra/workspace/common/utils/ControllerValidationUtils.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.workspace.common.utils;
+
+import bio.terra.workspace.common.exception.ValidationException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+
+public final class ControllerValidationUtils {
+
+  private static final List<String> VALID_FILTER_CONTROLLED_OPTIONS =
+      Arrays.asList("controlled", "uncontrolled", "all");
```

Comment:
I think these should be in an ENUM. Then you can use the same strings here as in the logic later.